stylesheets/extra.css

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Can you please update license headers to 2018

Reply:
Fixed with 65340d10bd6bbfd2805ae68595c0bc8f16d26ec9